### ISSUE TITLE
Recommended changes resulting from automated audit

### ISSUE THREAD

**Issue by Jericho** [enkafan/Cake.Figlet#17]
We performed an automated audit of your Cake addin and found that it does not follow all the best practices.

We encourage you to make the following modifications:

- [x] You are currently referencing Cake.Core 0.26.0. Please upgrade to 0.28.0
- [x] The Cake.Core reference should be private. Specifically, your addin's `.csproj` should have a line similar to this: `<PackageReference Include="Cake.Core" Version="0.28.0" PrivateAssets="All" />`

Apologies if this is already being worked on, or if there are existing open issues, this issue was created based on what is currently published for this package on NuGet.org and in the project on github.

**Comment by secana** [comment]
I'm running into the warning:
```
The assembly 'Cake.Figlet, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null'
is referencing an older version of Cake.Core (0.26.0).
For best compatibility it should target Cake.Core version 0.28.0.
```
I guess, this ticket would resolve the problem.

**Comment by pascalberger** [comment]
`Cake.Core` reference was fixed with #25

**Comment by pascalberger** [comment]
Build against `Cake.Core` 0.28.0 was fixed with #27
